packages/hash/shared/src/blocks.ts

```diff
@@ -233,7 +233,7 @@ export const fetchBlock = async (
 
 const textBlocks = new Set([
   "https://blockprotocol.org/blocks/@hash/paragraph",
-  "https://blockprotocol.org/blocks/@hash/header",
+  "http://localhost:63282",
```

Comment:
This will cause the deployed HASH to crash, since this URL doesn't exist there.

It might be better to split this PR out as follows:
1. PR in HASH to update the header block
2. PR in BP to publish the new header block
3. Rest of this PR

Alternatively, we can do everything at once but very quickly so that we minimise the time the deployed version is not working for. But since we're going to have to come back and change this line immediately anyway if we keep it like this, it's probably the same number of PRs overall.

Reply:
This was a mistake. My bad for rushing this out last night. Removed it in https://github.com/hashintel/hash/commit/d48a3a58ec97f805cd5e2f657995022a10dc822e 

As far sequencing, I think we should just merge it. HASH still supports `editableRef`, so it won't break anything to merge this before the header block is updated in the block hub.